test/mocha/10-api.js

```diff
@@ -0,0 +1,14 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {agent, httpsAgent} = require('bedrock-https-agent');
```

Comment:
I believe this is only working because it runs after `bedrock.init`.

Reply:
We may have to do something like this: https://github.com/digitalbazaar/bedrock-tokenization/blob/master/test/mocha/helpers.js#L52-L55

To test this.